src/io/files.js

```diff
@@ -869,7 +877,17 @@ p5.prototype.httpDo = function () {
       if (typeof a === 'string') {
         if (a === 'GET' || a === 'POST' || a === 'PUT' || a === 'DELETE') {
           method = a;
-        } else if(a === 'json' || a === 'jsonp' || a === 'xml' || a === 'text') {
```

Comment:
This change should be reverted as well as this also concerns return data from the server. If line length is too long, break the line at `||` (double pipe), breaking it into multiple `if`s may have performance penalties.